client/components/jetpack/daily-backup-status/test/index.jsx

```diff
@@ -0,0 +1,110 @@
+/**
+ * @jest-environment jsdom
+ */
+
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { shallow } from 'enzyme';
+import moment from 'moment';
+
+/**
+ * Internal dependencies
+ */
+import DailyBackupStatus from '..';
+
+import BackupFailed from '../status-card/backup-failed';
+import BackupScheduled from '../status-card/backup-scheduled';
+import BackupSuccessful from '../status-card/backup-successful';
+import NoBackupsYet from '../status-card/no-backups-yet';
+import NoBackupsOnSelectedDate from '../status-card/no-backups-on-selected-date';
+
+/**
+ * Mock dependencies
+ */
+jest.mock( 'react-redux', () => ( {
+	...jest.requireActual( 'react-redux' ),
+	useSelector: jest.fn().mockImplementation( ( selector ) => selector() ),
+} ) );
+
+jest.mock( 'state/ui/selectors/get-selected-site-id' );
+jest.mock( 'state/selectors/get-site-timezone-value' );
+jest.mock( 'state/selectors/get-site-gmt-offset' );
+
+jest.mock( 'state/selectors/get-rewind-capabilities' );
+import getRewindCapabilities from 'state/selectors/get-rewind-capabilities';
+
+jest.mock( 'lib/jetpack/backup-utils' );
+import { isSuccessfulDailyBackup, isSuccessfulRealtimeBackup } from 'lib/jetpack/backup-utils';
+
+describe( 'DailyBackupStatus', () => {
+	const getStatus = ( element ) => shallow( element ).find( 'DailyBackupStatus' ).dive();
+
+	beforeAll( () => {
+		// NOTE: We don't care about site ID, timezone, or GMT offset in this test suite;
+		// they're only important to child components or other dependencies.
+		// They're mocked in the "Mock dependencies" block above,
+		// but we allow them to use the default mock implementation,
+		// which always returns `undefined`.
+	} );
+
+	beforeEach( () => {
+		getRewindCapabilities.mockReset();
```

Comment:
I'd like to use `jest.resetAllMocks()` here, but `useSelector` is also a mock, so I needed a different approach.